corehq/apps/cloudcare/templates/formplayer/query_view.html

```diff
@@ -40,7 +40,20 @@ <h2 tabindex="0"><%- title %></h2>
   </form>
 </script>
 
+<script type="text/template" id="query-view-group-template">
+  <td>
+    <div data-toggle="collapse" data-target="#group-<%- groupKey %>" aria-expanded="true" aria-controls="group-<%- groupKey %>">
+      <label><%- groupName %></label>
```

Comment:
I don't know if `<label>` is relevant inside a div, as far as I know it's only for labeling an input element. And it already serves as the "label" for a clickable div by being contained inside it.

Reply:
I mainly used it for styling since the fields themselves use it too.